Extensions/Physics2Behavior/physics2runtimebehavior.js

```diff
@@ -269,10 +272,17 @@ gdjs.Physics2RuntimeBehavior.prototype.ownerRemovedFromScene = function() {
   this.onDeActivate();
 };
 
-gdjs.Physics2RuntimeBehavior.prototype.getPolygon = function(verticesData) {
+gdjs.Physics2RuntimeBehavior.getPolygon = function(verticesData) {
   var polygon = new gdjs.Polygon();
+
+  if (!verticesData) return polygon;
```

Comment:
Can you move this before creating a new polygon (previous line `var polygon = new gdjs.Polygon();`)?

Reply:
return `null` then :+1: